public/app/features/explore/PrometheusListView/RawListItemAttributes.tsx

```diff
@@ -0,0 +1,60 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/';
+import { useStyles2 } from '@grafana/ui/';
+
+import { RawListValue } from './RawListItem';
+
+const getStyles = (theme: GrafanaTheme2) => {
+  // Borrowed from the monaco styles
+  const reddish = theme.isDark ? '#ce9178' : '#a31515';
+  const greenish = theme.isDark ? '#73bf69' : '#56a64b';
+
+  return {
+    metricName: css`
+      color: ${greenish};
+    `,
+    metricValue: css`
+      color: ${reddish};
+    `,
+    expanded: css`
+      display: block;
+      text-indent: 1em;
+    `,
+    contracted: css``,
+  };
+};
+
+const RawListItemAttributes = ({
+  value,
+  index,
+  length,
+  isExpandedView,
+}: {
+  value: RawListValue;
+  index: number;
+  length: number;
+  isExpandedView: boolean;
+}) => {
+  const styles = useStyles2(getStyles);
+
+  // From the beginning of the string to the start of the `=`
+  const attributeName = value.key;
+
+  // From after the `="` to before the last `"`
+  const attributeValue = value.value;
+
+  return (
+    <span className={isExpandedView ? styles.expanded : styles.contracted} key={index}>
```

Comment:
```suggestion
    <span className={isExpandedView && styles.expanded} key={index}>
```

Reply:
This was throwing a ts error as this className type is `string | false`, I kept the ternary, but deleted that unnecessary contracted css class.

Reply:
i think you can do `<span className={cx(isExpandedView && styles.expanded)} key={index}>` instead :)